package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "website-lambda-meetup",
+  "version": "0.1.0",
+  "description": "Repository for Lambda functionality related to periodically fetching data from the meetup.com API.",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint *.js",
+    "run": "node src/index.js"
+  },
+  "repository": "https://github.com/ProvidenceGeeks/website-lambda-meetup.git",
+  "author": "mdiwale@kenzan.com",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^4.8.0"
+  },
+  "dependencies": {
+    "express": "^4.16.1"
+  },
+  "eslintConfig": {
```

Comment:
please use a  _.eslintrc_ file, like in [**website-frontend**](https://github.com/ProvidenceGeeks/website-frontend/blob/master/.eslintrc).